doc/user/content/free-trial-faqs.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Free trial FAQs"
+description: "Answers to frequency asked questions about Materialize free trials"
+menu:
+  main:
+    parent: "quickstarts"
+    weight: 15
+---
+
+When you [sign up for Materialize](https://materialize.com/register/), you get a free trial account so you can explore the product without worrying about initial costs. This page answers some frequently asked questions about free trials.
+
+## What's included in a free trial?
+
+In Materialize, [cluster replicas](/overview/key-concepts/#cluster-replicas) are the physical resources for doing computational work. Cluster replicas come in various sizes, and each size corresponds to a specific number of [compute credits per hour](https://materialize.com/pricing/).
+
+During your free trial, your rate of credit consumption in a given region cannot exceed 4 credits per hour at any point in time. This limit should accommodate most trial scenarios. If you need additional resources during your trial, please contact sales@materialize.com.
+
+## How long does a free trial last?
+
+14 days.
+
+If you want to continue using Materialize beyond 14 days, please contact sales@materialize.com. Otherwise, Materialize will delete your resources and data at the end of the trial period.
+
+## How do I monitor credit consumption?
+
+To see your current rate of credit consumption, measured in credits per hour, run the following query:
+
+```sql
+SELECT sum(s.credits_per_hour) AS credit_consumption_rate
+FROM mz_cluster_replicas r
+JOIN mz_internal.mz_cluster_replica_sizes s ON r.size = s.size;To
+```
+
+For example, if you start your free trial with the [getting started guide](/get-started/), you'll end up consuming credits at a rate of `3.5` per hour:
+
+```nofmt
+ total_credits_per_hour
+------------------------
+                    3.5
+(1 row)
+```
+
+## Can I go over the credit rate limit?
+
+No, you cannot go over the rate limite of 4 credits per hour at any time during your free trial. If you try to add a replica that puts you over the limit, Materialize will return an error.
+
+If you need additional credits during your trial, please contact sales@materialize.com.
+
+## How do I get help during my trial?
+
+If you have questions about Materialize or need support, reach out to us in our [Community Slack](https://materialize.com/s/chat) or set up time to [chat with our team](http://materialize.com/convert-account/).
+
+## How do I convert to a paid plan?
```

Comment:
This also feels a little redundant.

Reply:
Same - we mention connecting with sales multiple times. I think we can swap those `mailto:` links out for the convert account landing page and then remove a few mentions of contacting sales

Reply:
Hmm. I'm not exactly sure what changes to make here. Will think it through.

Reply:
Made some changes.